js/lib/beautify-html.js

```diff
@@ -84,7 +84,7 @@
         return s.replace(/\s+$/g, '');
     }
 
-    function style_html(html_source, options, js_beautify, css_beautify) {
+    function style_html(html_source, options, js_beautify_raw, css_beautify_raw) {
```

Comment:
Instead of changing this, why not use different names for the `js-beautify` and `css-beautify` vars. 😄